CONTRIBUTING.md

```diff
@@ -467,6 +467,14 @@ except (ImportError, IOError) as err:
 This rule doesn't apply to the test files since these files are loaded only when
 running tests and in such a case your module and their external dependencies are installed.
 
+**You also need to test your code without the python lib nor the module installed** 
```

Comment:
Currently we have a similar check from https://github.com/oca/pylint-odoo

 - W7935 | External dependency "%s" without ImportError. More info: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#external-dependencies | missing-import-error
 - W7936 | Missing external dependency "%s" from manifest. More info: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#external-dependencies | missing-manifest-dependency

If you use a `import` of a external library that odoo don't have defined in requirements pylint-odoo will emit a message to add a `try-except-ImportError` and a adds `external dependency` to manifest file